src/node/nodestate.h

```diff
@@ -360,7 +294,7 @@ namespace ccf
           // network
           open_member_frontend();
 
-          if (!create_and_send_request(args, quote))
+          if (!create_and_send_request(args, quote, key_share_info))
```

Comment:
I'd forgotten that the first entry was executed by the member frontend. So we generate the encrypted ledger secrets and shares here and pass it to the frontend for execution. This is safe for PBFT as the request recorded in the ledger only contains public info.